src/observability/summarizer.go

```diff
@@ -73,6 +73,29 @@ func extractNetworkInfoFromSystemLog(netLog pb.Log, pods []types.Pod, services [
 	return ip, destNs, destLabel, port, bindPort, bindAddress, protocol, nwrule, nil
 }
 
+func extractSyscallInfoFromSystemLog(sysCallLog pb.Log, pods []types.Pod, services []types.Service) (string, string, string, string, error) {
+	var ParentProcess, ChildProcess, Syscall, Parameters string = "", "", "", ""
+	err := errors.New("not a valid Syscall")
+
+	if sysCallLog.Operation == types.SumOpTypeSyscall {
+		ParentProcess = sysCallLog.ParentProcessName
+		ChildProcess = sysCallLog.ProcessName
+
+		resslice := strings.SplitN(sysCallLog.Data, " ", 2)
+		if strings.Contains(resslice[0], "syscall") {
+			Syscall = strings.Split(resslice[0], "=")[1]
+		}
+		if len(resslice) >= 2 {
+			Parameters = resslice[1]
+		}
+
+	} else {
+		return "", "", "", "", err
+	}
```

Comment:
```suggestion
	if sysCallLog.Operation != types.SumOpTypeSyscall {
		return "", "", "", "", err
	}
	ParentProcess = sysCallLog.ParentProcessName
	ChildProcess = sysCallLog.ProcessName

	resslice := strings.SplitN(sysCallLog.Data, " ", 2)
	if strings.Contains(resslice[0], "syscall") {
		Syscall = strings.Split(resslice[0], "=")[1]
	}
	if len(resslice) >= 2 {
		Parameters = resslice[1]
	}
```

Reply:
Thank you for your suggestions. Changes are commited.